vllm/model_executor/models/opt.py

```diff
@@ -351,3 +351,9 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                 weight_loader = getattr(param, "weight_loader",
                                         default_weight_loader)
                 weight_loader(param, loaded_weight)
+
+        # memory optimization: if lm_head quantization is not enabled lm_head
+        # is same as embed_tokens. keep only one copy
+        if (self.quant_config is None
+                or not self.quant_config.lm_head_quantized):
+            self.lm_head = self.model.decoder.embed_tokens
```

Comment:
@robertgshaw2-neuralmagic  I moved lm_head_quantized property into base `QuantizationConfig` so that this code here doesn't have nasty isinstance checks for all quant configs that suppor t`lm_head` quants. Most likely other quant models/code will also need this check.